policy/lib/main_denies.rego

```diff
@@ -1,5 +1,7 @@
 package lib
 
+import data.lib.time as time_lib
```

Comment:
Why don't we put the time based policy refactor in a separate PR?

Reply:
Good idea: https://github.com/hacbs-contract/ec-policies/pull/60